ivy_tests/test_ivy/test_frontends/test_numpy/test_sorting_searching_counting/test_searching.py

```diff
@@ -343,3 +343,45 @@ def test_numpy_nanargmin(
         axis=axis,
         keepdims=keep_dims,
     )
+
+
+@st.composite
+def _extract_strategy(draw):
+    dtype_and_cond = draw(
+        helpers.dtype_and_values(
+            available_dtypes=helpers.get_dtypes("valid"),
+        )
+    )
+    dtype_and_arr = draw(
+        helpers.dtype_and_values(
+            available_dtypes=helpers.get_dtypes("valid"),
+        )
+    )
+    return dtype_and_cond, dtype_and_arr
+
+
+# extract
+@handle_frontend_test(
+    fn_tree="numpy.extract",
+    dtype_and_x=_extract_strategy(),
```

Comment:
you havent defined what to do with  _extract_strategy ( ), this makes your tests to fail.

Reply:
Hello, 

Can you clarify which of my tests fails? 
Because my ivy frontend test for the extract method works fine. I ran it many times. This is the only test that I implemented.

And for my extract strategy, it is just a strategy for data generation. 
I built it with the decorator "st composite".

As the doc says:
"An initialized strategy is an object that is used by Hypothesis to generate data for the test."

Someone did the same with the ivy frontend test for the where method. He built a strategy called "_broadcastable_trio" and used it for data generation.

Thank you

Reply:
@ndurumo254

Reply:
Hi,
I think the previous failures were not related to my ivy frontend test... but I think I corrected them all now and did some new commits.

thanks